scripts/wireguard/disableCONF.sh

```diff
@@ -106,7 +106,7 @@ for CLIENT_NAME in "${CLIENTS_TO_CHANGE[@]}"; do
       read -r -p "Confirm you want to disable ${CLIENT_NAME}? [Y/n] "
     fi
 
-    if [[ "${REPLY}" =~ ^[Yy]$ ]]; then
+    if [[ "${REPLY}" =~ ^[yY]$ ]] || [[ -z "${REPLY}" ]]; then
```

Comment:
Why inverting `y` and `Y` though?

Reply:
Mostly because I incorrectly reverted a change I made, I can switch it back

Reply:
Fixed in 90ba2d1